locales/nl/json.json

```diff
@@ -1071,69 +1071,69 @@
     "Yes, I'm Sure": "Ja ik weet het zeker",
     "Yes, Leave": "Ja, vertrek",
     "Yes, Remove": "Ja, verwijderen",
-    "You": "U",
+    "You": "Je",
     "You are already subscribed.": "Je bent al ingeschreven.",
-    "You are currently subscribed to the :planName plan.": "U bent momenteel geabonneerd op het :planName-abonnement.",
+    "You are currently subscribed to the :planName plan.": "Je bent momenteel geabonneerd op het :planName-abonnement.",
     "You are currently within your free trial period. Your trial will expire on :date.": "Je maakt nu gebruik van een gratis proefabonnement. Je proefabonnement verloopt op :date.",
-    "You are currently within your free trial period. Your trial will expire on :date. Starting a new subscription will end your trial.": "U bevindt zich momenteel in uw gratis proefperiode. Uw proefperiode verloopt op :date. Als u een nieuw abonnement start, wordt uw proefperiode beëindigd.",
+    "You are currently within your free trial period. Your trial will expire on :date. Starting a new subscription will end your trial.": "Je bevindt zich momenteel in je gratis proefperiode. Je proefperiode verloopt op :date. Als je een nieuw abonnement start, wordt je proefperiode beëindigd.",
```

Comment:
"Je bevindt zich momenteel in je gratis proefperiode." is incorrect. It should be "Je bevindt **je** momenteel in je gratis proefperiode."

Reply:
Fixed in https://github.com/Laravel-Lang/lang/pull/3255.